ui/app/templates/components/namespace-picker.hbs

```diff
@@ -1,110 +1,84 @@
-{{#if (and (not this.accessibleNamespaces.length) this.inRootNamespace)}}
-  <div class="namespace-picker no-namespaces">
-    {{! Just yield the logo if they're in the root namespace and only have access to it }}
-    {{yield}}
-  </div>
```

Comment:
The namespace picker is now always displayed. Namespaces the users does not have access too will not show up in the menu.

Reply:
I updated the default state to display as `root`.